addons/mail/static/src/components/discuss_mobile_mailbox_selection_item/discuss_mobile_mailbox_selection_item.js

```diff
@@ -0,0 +1,37 @@
+/** @odoo-module **/
+
+import { registerMessagingComponent } from '@mail/utils/messaging_component';
+
+const { Component } = owl;
+
+export class DiscussMobileMailboxSelectionItem extends Component {
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+     * @returns {DiscussView}
+     */
+    get discussView() {
+        return this.messaging && this.messaging.models['DiscussView'].get(this.props.localId);
+    }
+
+    /**
+     * @returns {Thread}
+     */
+    get mailbox() {
+        return this.messaging && this.messaging.models['Thread'].get(this.props.mailboxLocalId);
+    }
+
+}
+
+Object.assign(DiscussMobileMailboxSelectionItem, {
+    props: {
+        localId: String,
```

Comment:
Doesn't it feel a bit weird to have a property called `localId` that is the local ID of DiscussView while the name of the component is `DiscussMobileMailboxSelectionItem`?

Reply:
I did not want to have a prop name that is too long, and using `localId` feels sort of like an arbitrary component split.
Follow-up PRs will introduce the model view, and then pass only `localId` as the record of this specific component.

Reply:
Changes done